movies-app/src/components/ConfirmModal/ConfirmModal.jsx

```diff
@@ -0,0 +1,37 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { I18N } from '../../core/i18n';
+import { LocaleContext } from '../../core/i18n/locale';
+import { ReactPortal } from '../ReactPortal';
+import CloseIcon from '../../core/constants/close-icon.svg';
+import './ConfirmModal.scss';
+
+export const ConfirmModal = ({ header, question, closeModal }) => {
+  const locale = useContext(LocaleContext);
+  return (
+    <ReactPortal>
+      <div className="confirm-modal">
+        <div className="modal-close">
```

Comment:
Same optional suggestion as for `Header` components - grouping JSX with functions tends to improve readability going forward.